### PR TITLE
IDE-2100 nested project in the workspace SDK shows wrong action

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

File: tools/plugins/com.liferay.ide.project.ui/plugin.xml [hunk 1]
```diff
@@ -303,7 +303,7 @@
             id="com.liferay.ide.project.ui.existingLiferayProject"
             objectClass="org.eclipse.core.resources.IFolder">
          <action
-               class="com.liferay.ide.project.ui.action.ImportLiferayProjectAction"
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/plugin.xml]: Can you delete the old class, it isn't used any more.

File: tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/action/ImportSDKProjectAction.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+ */
+public class ImportSDKProjectAction implements IObjectActionDelegate
+{
+    private ISelection fSelection;
+
+    public ImportSDKProjectAction()
+    {
+    }
+
+    public Display getDisplay()
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/action/ImportSDKProjectAction.java]: No need for this method, just call Display.getDefault()

File: tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/action/ImportSDKProjectAction.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            }
+
+            SDKProjectImportWizard wizard =
+                new SDKProjectImportWizard(  projectLocation );
+
+            final Display display = getDisplay();
+
+            final WizardDialog dialog = new WizardDialog( display.getActiveShell(), wizard );
+
+            BusyIndicator.showWhile
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/action/ImportSDKProjectAction.java]: No need for BusyIndicator, just call dialog.open()

**Comment by gamerson** [comment]
See comments and please resend, thanks!

File: tools/plugins/com.liferay.ide.project.ui/plugin.xml (hunk 1 above)
Comment (simonjhy) [inline tools/plugins/com.liferay.ide.project.ui/plugin.xml]: This has been deleted in other ticket.

Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/plugin.xml]: 怪不得

**Pull request closed without merging**